README.md

```diff
@@ -271,10 +272,15 @@ Companies that have allowed us to speak about our work can be found here. Many m
 
 | Product | Review Date | Level of Effort | Announcement |
 | --- | --- | --- | --- |
+| [Redjack](https://www.redjack.com/) | Dec 2021 | 2 person-weeks | |
+| [DeGate](https://degate.com/?en-US) | Dec 2021 | 4 person-weeks | |
+| [Cardstack](https://cardstack.com/) | Dec 2021 | 4 person-weeks | |
+| [Frax](https://frax.finance/) | Dec 2021 | 4 person-weeks | |
+| [Sherlock](https://www.sherlock.global/) | Dec 2021 | 4 person-weeks | |
```

Comment:
im pretty sure this is not the sherlock y'all did an audit for lol

Reply:
i think you mean https://www.sherlock.xyz/

Reply:
ugh, yes